meta-xt-prod-devel-rcar-control/recipes-guests/domd/domd.bbappend

```diff
@@ -24,10 +29,13 @@ do_install:append() {
     # Install domd-set-root script
     install -d ${D}${libdir}/xen/bin
     install -m 0744 ${WORKDIR}/domd-set-root ${D}${libdir}/xen/bin
+    install -d ${D}${sysconfdir}/systemd/system/domd.service.d
 
-    # Call domd-set-root script
-    echo "[Service]" >> ${D}${systemd_unitdir}/system/domd.service
-    echo "ExecStartPre=${libdir}/xen/bin/domd-set-root" >> ${D}${systemd_unitdir}/system/domd.service
+    if ${@bb.utils.contains('DISTRO_FEATURES', 'enable_virtio', 'true', 'false', d)}; then
+        install -m 0644 ${WORKDIR}/domd-set-root-virtio.conf ${D}${sysconfdir}/systemd/system/domd.service.d
```

Comment:
Could you introduce `DOMD_SET_ROOT_CONF` variable which holds config name, instead of covering whole receipt with `if`s and `bb.contains`?

Reply:
this patch can be simplified without even touching domd-set-root